pyswarms/utils/plotters/formatters.py

```diff
@@ -10,6 +10,7 @@
 import numpy as np
 from attr import attrs, attrib
 from attr.validators import instance_of
+from matplotlib import cm
 
```

Comment:
I think the best way to validate the colormaps is to check if they are instances of the `matplotlib.colors.Colormap` class. Both `ListedColormap` (the type of `cm.viridis`) and `LinearSegmentedColormap` (the type of `cm.binary`) are subclasses of `Colormap`.

So my suggestion is:

```python
from matplotlib import cm, colors

class Designer(object):
    colormap = attrib(
        validator=instance_of(colors.Colormap),
        default=cm.viridis,
     )
```

Can you confirm if this works? :smile: 
If this works, I think there's no need to write tests for all colormaps since we have a validator that gives out an error whenever we enter a non-`Colormap`

Reply:
That makes sense! I had been trying to validate colors.Colormap with type(), but that's redundant with attrib and messes things up.